test/test_address_book.py

```diff
@@ -120,6 +121,30 @@ def test_unparsable_files_can_be_skipped(self):
             'WARNING:khard.address_book:1 of 1 vCard files of address book '
             'test could not be parsed.')
 
+    def test_env_var_paths(self):
+        var_name = "KHARD_FOO"
+
+        # Unset env vars shouldn't expand.
+        if var_name in os.environ:
+            os.environ.pop(var_name)
```

Comment:
I think it is better to use `unittest.mock.patch` to mock environment variables during tests. There are some examples of this in the tests.